composer.json

```diff
@@ -7,16 +7,16 @@
         }
     },
     "require": {
-        "php": "^7.1 || ^8.0",
+        "php": "^7.4 || ^8.0",
         "paysera/lib-serializer": "^3.0",
         "doctrine/orm": "^2.0",
-        "symfony/config": "^3.0 || ^4.0",
-        "symfony/dependency-injection": "^3.0 || ^4.0",
-        "symfony/validator": "^3.0 || ^4.0",
-        "symfony/security-core": "^3.0 || ^4.0",
-        "symfony/routing": "^3.0 || ^4.0",
-        "symfony/http-foundation": "^3.0 || ^4.0",
-        "symfony/http-kernel": "^3.0 || ^4.0"
+        "symfony/config": "^5.0",
```

Comment:
No BC? Should be `^3.0 || ^4.0 || ^5.0` I guess.

Reply:
No, several classes/namespaces have been changed.

Reply:
@tomasliubinas For example those events. The idea was to have a new major tag 6.0.0 with a separate branch. So every version of symfony will be supported
![image](https://user-images.githubusercontent.com/131874317/234587049-a4909c3a-a51e-431f-a8b1-c563153f598d.png)

Reply:
https://github.com/symfony/http-kernel/blob/v5.0.0/CHANGELOG.md#500

```
removed GetResponseEvent, use RequestEvent instead
removed GetResponseForControllerResultEvent, use ViewEvent instead
removed GetResponseForExceptionEvent, use ExceptionEvent instead
removed PostResponseEvent, use TerminateEvent instead
```

Reply:
FYI this library should not be used for new projects, new APIs should be written with https://github.com/paysera/lib-api-bundle

Consider replacing this lib in your project with https://github.com/paysera/lib-api-bundle if it makes sense, eg. when project has small number of controllers